README.md

```diff
@@ -172,6 +172,47 @@ For invoking Cloud Identity-Aware Proxy, you will need to pass the Client ID
 used when you set up your protected resource as the target audience. See how to
 [secure your IAP app with signed headers](https://cloud.google.com/iap/docs/signed-headers-howto).
 
+#### Call using a specific JSON key
+If you want to use a specific JSON key instead of using `GOOGLE_APPLICATION_CREDENTIALS` environment variable, Yo can
+ do this:
+ 
+```php
+use Google\Auth\CredentialsLoader;
+use Google\Auth\Middleware\AuthTokenMiddleware;
+use GuzzleHttp\HandlerStack;
+
```

Comment:
@imdhemy  @bshaffer  I suggest adding a use statement for `Client`.

```
$client = new Client([
  'handler' => $stack,
  'base_uri' => 'https://www.googleapis.com',
  'auth' => 'google_auth'  // authorize all requests
]);
```

Or is it more common to use the following?
https://github.com/googleapis/google-auth-library-php/pull/324#issuecomment-793060136

```
// Using credentials option
$client = new Google\Client(['credentials' => $jsonKey]);
// Using setAuthConfig
$client = new Google\Client();
$client->setAuthConfig($jsonKey);
```

Reply:
`Client` here is `GuzzleHttp\Client`